srcpkgs/mattermost-desktop/template

```diff
@@ -0,0 +1,37 @@
+# Template file for 'mattermost-desktop'
+pkgname=mattermost-desktop
+version=4.5.2
+revision=1
+archs="x86_64 i686"  # no musl due to electron
```

Comment:
Remove that line as it is not a package constraint but dependency issue.

Reply:
I removed the comment, although one could argue that since Electron is an 'implicit' dependency (i.e. not listed in any of the `*depends`, but downloaded during `do_build`), the reason behind the restriction wouldn't be immediately clear.